notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -1281,6 +1281,84 @@
+      "from google.cloud import aiplatform_v1beta1\n",
+      "from google.api_core import operations_v1\n",
+      "from google.cloud.aiplatform_v1beta1 import FeaturestoreOnlineServingServiceClient\n",
+      "from google.cloud.aiplatform_v1beta1 import FeaturestoreServiceClient\n",
+      "from google.cloud.aiplatform_v1beta1.types import featurestore_online_service as featurestore_online_service_pb2\n",
+      "from google.cloud.aiplatform_v1beta1.types import types as types_pb2\n",
+      "from google.protobuf import field_mask_pb2\n",
+      "\n",
+      "API_ENDPOINT = \"{}-aiplatform.googleapis.com\".format(REGION)\n",
+      "# Create client connection\n",
+      "admin_client = FeaturestoreServiceClient(\n",
+      "    client_options={\"api_endpoint\": API_ENDPOINT})\n",
+      "data_client = FeaturestoreOnlineServingServiceClient(client_options={\"api_endpoint\": API_ENDPOINT})"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "execution_count": null,
+     "metadata": {},
+     "outputs": [],
+     "source": [
+      "# Call write_feature_values to ingest data to users entity_type. \n",
+      "data_client.write_feature_values(\n",
+      "    entity_type = admin_client.entity_type_path(PROJECT_ID, REGION, FEATURESTORE_ID, \"users\"),\n",
+      "    payloads = [\n",
+      "        featurestore_online_service_pb2.WriteFeatureValuesPayload(\n",
+      "            entity_id = \"1305\",\n",
+      "            feature_values = {\n",
+      "                \"age\" : featurestore_online_service_pb2.FeatureValue(\n",
+      "                    int64_value = 34),\n",
+      "                \"gender\" : featurestore_online_service_pb2.FeatureValue(\n",
+      "                   string_value = \"female\"),\n",
+      "                \"liked_genres\" : featurestore_online_service_pb2.FeatureValue(\n",
+      "                   string_array_value= types_pb2.StringArray(\n",
+      "                        values = [\"drama\", \"action\"]))\n",
+      "            }),\n",
+      "        featurestore_online_service_pb2.WriteFeatureValuesPayload(\n",
+      "            entity_id = \"1306\",\n",
+      "            feature_values = {\n",
+      "                \"age\" : featurestore_online_service_pb2.FeatureValue(\n",
+      "                    int64_value = 50),\n",
+      "                \"gender\" : featurestore_online_service_pb2.FeatureValue(\n",
+      "                   string_value = \"male\"),\n",
+      "                \"liked_genres\" : featurestore_online_service_pb2.FeatureValue(\n",
+      "                   string_array_value= types_pb2.StringArray(\n",
+      "                        values = [\"suspense\", \"comedy\"]))\n",
+      "            }),\n",
+      "    ])"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "execution_count": null,
+     "metadata": {},
+     "outputs": [],
+     "source": [
+      "When `write_feature_values` API succeeded, it returns empty response.\n",
```

Comment:
Upon successful completion, the `write_feature_values` API returns an empty response.

Reply:
We return error code and error message when backend failed to write to storages